src/unix.rs

```diff
@@ -435,6 +436,14 @@ unsafe fn fd_check(fd: c_int, check_pipe: bool) -> Result<(), FromEnvErrorInner>
     }
 }
 
+fn clone_fd_and_set_cloexec(fd: c_int) -> Result<File, FromEnvErrorInner> {
```

Comment:
Does this really set `FD_CLOEXEC`?

Reply:
Yes, it `File::try_clone` calls [`BorrowedFd<'_>::try_clone_to_owned`](https://doc.rust-lang.org/src/std/os/fd/owned.rs.html#96-113) which sets close-on-exec.